src/server/common/helpers/redis-client.js

```diff
@@ -21,7 +21,8 @@ function buildRedisClient() {
     redisClient = new IoRedis({
       port,
       host: config.get('redisHost'),
-      db
+      db,
+      keyPrefix: config.get('redisKeyPrefix')
```

Comment:
This is used in multiple places in the same file. Can we do `const keyPrefix = config.get('redisKeyPrefix')` and then just reference `keyPrefix` later on in the objects?

Reply:
updated